data/remnant/remnant missions.txt

```diff
@@ -2245,6 +2245,417 @@ mission "Remnant: Face to Maw 2B"
+	landing
+	source "Newhome"
+	destination "Viminal"
+	cargo "Keystones" 19
+	to offer
+		has "Remnant: Crystal Research 3: done"
+	on accept
+		dialog
+			`As you were reading the previous message, a freight hauler arrives with the last few crates and starts unloading them. Looking over the assembled stack, you note that each crate bears the logos and advertisements for a different organization, shipper, or enterprise. The second message turns out to be a confirmation that you have picked up 19 keystones. While there are no new instructions, the subtext is clear: Time to head back.`
+	on complete
+		payment 2000000
+		conversation
+			`As you descend through the clouds to Viminal, you get the feeling someone is watching you, but your sensors show nothing more than a smattering of Gulls and a patrol squad in the distance. Looking down, you spot an extra battery of anti-air artillery next to the main entrance to the spaceport. You note the artillery tracking you and quickly tap in the recognition code to the IFF system. Within seconds, the large guns are swinging back to covering the initial approach.`
+			`	Passing through the cavernous opening into the great spaceport dome, there is a shimmer in the air next to you and a Starling fades into view, settling onto the platform beside you. The pilot signs a quick, "Greetings, anticipating talking with you," before heading back toward their lock. Behind them, the Starling continues its shutdown checks without any supervision at all.`
+			`	As you finish your own shutdown routine, your commlink informs you that your account has been updated with <payment>.`
+
+
+
+ship "Starling" "Starling (Keystone)"
+	outfits
+		"Inhibitor Cannon" 3
+		"Point Defense Turret"
+		
+		"Epoch Cell"
+		"Crystal Capacitor"
+		"Emergency Ramscoop" 2
+		"Quantum Keystone"
+		"Salvage Scanner"
+		
+		"Forge-Class Thruster"
+		"Forge-Class Steering"
+		"Crucible-Class Steering"
+		"Hyperdrive"
+
+
+
+mission "Remnant: Crystal Research 5"
+	name "Scan the <npc> in <waypoints>"
+	description `Go scan the <npc> to record the keystone post-wormhole.`
+	landing
+	waypoint "Peragenor"
+	source "Viminal"
+	destination "Viminal"
+	to offer
+		has "Remnant: Crystal Research 4: done"
+	on offer
+		conversation
+			`You exit the ship to find that the pilot is actually Aeolus, who has already marshaled some dock workers with a number of wagons and lifters to unload the cargo.`
+			`	"Ah, <first>, thank you for retrieving all these keystones." He sweeps some kind of a device over the crates, then opens one. He pokes at the contents, and pulls out a brochure with a picture of a quantum keystone and a pair of respectable looking Hai. "What's this?" He asks curiously. "Because I bought a keystone, they think I might need more? Our histories told us that humanity thrived on the pursuit of wealth, it just seems inefficient to me."`
```

Comment:
Changed the last sentence from 

> "Because I bought a keystone, they think I might need more? Our histories told us that humanity thrived on the pursuit of wealth, it just seems inefficient to me."

to

> "Because I bought a keystone, they think I might need more? That seems like a decidedly inefficient way of promoting their product."

More of a direct and relevant statement, less philosophizing about humanity's economics. Which actually aren't even really relevant considering it is a *Hai* source, not a *Human* source.

Reply:
Seems good.